ui/app/src/context/plugin-registry/registry-state.ts

```diff
@@ -21,6 +21,7 @@ import {
   PluginType,
   AnyPluginImplementation,
   JsonObject,
+  ALL_PLUGIN_TYPES,
```

Comment:
This change to have a runtime list of all plugin types simplifies some of the registry code below so that now there are fewer places to go make changes whenever we add a new plugin type.